### DISCUSSION TITLE
Feature Freeze for 1.1

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#347] [Development]
Hi @zhuiks, @augustin-colesnic,

It's two weeks until the release. Let's try to get PR #317, PR #316 and  PR #345 merged soon ... after that I will raise the usual translation issue, so that translators have some time to update the locales.

This also means that after these merges we shouldn't have any feature changes anymore for 1.1. Bugfixes and Refactoring are OK, though.

**Comment by zhuiks** [comment]
If #345 doesn't break anything on Android it is ready to be merged

**Reply by tobias-klein** [reply]
Yes, I'll test it this weekend and then merge if fine.

**Comment by augustin-colesnic** [comment]
PR #316 is ready for merge. I did all the suggested changes. Please check.

**Reply by tobias-klein** [reply]
Thanks. I'll review.

**Comment by zhuiks** [comment]
@tobias-klein 
I think #317 PR won't make it to the June realize. It's a big refactoring. I finished basic functionality, but I prefer to leave it for the next release in August, to polish it up.

But #351 is ready to be merged. After that there should not be any updates to locale files, so translators can be asked to do the translation.

I'm planning to finish refactor the layout #302. But there should not be any locale/feature changes
